SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -345,6 +345,16 @@ SPATIALGDKEDITOR_API void GenerateSchemaForSublevels(const FString& SchemaOutput
 	Writer.WriteToFile(FString::Printf(TEXT("%sSublevels/sublevels.schema"), *SchemaOutputPath));
 }
 
+SPATIALGDKEDITOR_API void GenerateSchemaForRPCEndpoints()
```

Comment:
Nit: instead of having two functions `GenerateSchemaForRPCEndpoints` with different signature, I'd have one with default value "". And then check for an empty string argument inside the function.

Reply:
I considered this approach as well, but decided to go with the overload in the end. We do the same thing here for `GenerateSchemaForSublevels` and I think it's a bit cleaner in that the no-arguments overload uses the defaults for all arguments and it's the version called by the toolbar and the commandlet, whereas the one with the explicit arguments is called by the default one and the tests.